modules/connected-to-disconnected-prepare-mirror.adoc

```diff
@@ -0,0 +1,99 @@
+= Preparing the cluster for mirroring 
+
+Before disconnecting your cluster, you must mirror, or copy, the images in the registries from which your cluster pulls images while your cluster is still connected. In order to mirror the images, you must prepare your cluster by:
+
+* Adding the mirror registry certificates to the list of trusted CAs on your host.
+* Creating a `.dockerconfigjson` file that contains your image pull secret, which is from `cloud.openshift.com" token, which you use to mirror the {product-title} registries. 
+* Creating an `ImageContentSourcePolicy` (ICSP) object that associates the mirrored repository to the source repository for pods. The ICSP redirects image pulls from online registries to your offline mirror registry.
+
+
+.Procedure
+
+. Configuring credentials that allow images to be mirrored:
+
+.. Add the registry CA certificate, in the simple PEM or DER file formats, to the list of trusted CAs. For example:
++
+[source,terminal]
+----
+$ cp ~/certificate-trust-examples/Cert-trust-test-ca.pem /usr/share/pki/ca-trust-source/anchors/
+----
++
+[source,terminal]
+----
+$ update-ca-trust
+----
+
+.. Extract the `.dockerconfigjson` file from the global pull secret:
++
+[source,terminal]
+----
+$ oc extract secret/pull-secret -n openshift-config --confirm --to=. 
+----
++
+.Example output
+[source,terminal]
+----
+.dockerconfigjson
+----
+
+.. Edit the `.dockerconfigjson` file to add your mirror registry and authentication credentials and save it as a new file:
+// copied from olm-accessing-images-private-registries
++
+[source,terminal]
+----
+$ cat .dockerconfigjson | \
+    jq --compact-output '.auths["<registry>:<port>/<namespace>/"] |= . + {"auth":"<token>"}' \//<1>
+    > new_dockerconfigjson
+----
+<1> Replace `<registry>:<port>/<namespace>` with the mirror registry details and `<token>` with your authentication credentials.
+
+. Create an `ImageContentSourcePolicy` file: 
+// copied from images-configuration-registry-mirror
+
+.. Create an `ImageContentSourcePolicy` custom resource, replacing the source and mirrors with your own registry and repository pairs and images:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: example
+spec:
+  repositoryDigestMirrors:
+    - 172.31.249.15:443/ocp/release <1>
+    source: quay.io/openshift-release-dev/ocp-release <2>
+  - mirrors:
+    - 172.31.249.15:443/ocp/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+----
+<1> Specifies the name of the image registry and repository.
+<2> Specifies the registry and repository containing the content that is mirrored.
+
+.. Create the new `ImageContentSourcePolicy` object:
++
+[source,terminal]
+----
+$ oc create -f registryrepomirror.yaml
+----
+
+. Optional. If you plan to use the new mirror to pull images for a future {product-title} disconnected installation, add the following section to the `install-config.yaml`:
```

Comment:
I don't think we need to mention this here as it is already in the install a disconnected cluster docs and this doc is talking about moving a connected installed cluster to disconnected.

Reply:
@mburke5678  I still think we should remove this from here as it doesn't relate to moving a connected cluster to disconnected.